tests/filecheck/dialects/irdl/cmath.irdl.mlir

```diff
@@ -0,0 +1,51 @@
+// RUN: xdsl-opt %s | xdsl-opt | filecheck %s
+
+builtin.module {
+  // CHECK-LABEL: irdl.dialect @cmath {
```

Comment:
I don't think CHECK-LABEL is supported by filecheck, right? [docu](https://filecheck.readthedocs.io/en/stable/index.html)